setup.py

```diff
@@ -70,9 +70,10 @@
     },
     # Packaging
     package_dir={"": "src"},
-    packages=find_packages("src") + namespace_packages,
```

Comment:
The `namespace_packages` list is still defined in this file. As it serves no purpose anymore, we can remove it.